actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -92,10 +98,28 @@ def redirect_back(fallback_location:, allow_other_host: true, **args)
     #
     # All other options that can be passed to #redirect_to are accepted as
     # options and the behavior is identical.
-    def redirect_back_or_to(fallback_location, allow_other_host: true, **args)
-      referer = request.headers["Referer"]
-      redirect_to_referer = referer && (allow_other_host || _url_host_allowed?(referer))
-      redirect_to redirect_to_referer ? referer : fallback_location, **args
+    def redirect_back_or_to(fallback_location, allow_other_host: allow_open_redirects, **args)
+      referer = request.headers.fetch("Referer", fallback_location)
+      response_options = {
+        fallback_location: allow_open_redirects ? fallback_location : nil,
+        allow_other_host: allow_other_host,
+        **args,
+      }
+      redirect_to referer, response_options
```

Comment:
Is this a Shopify style?

Our style is:

```ruby
redirect_to referer, fallback_location: fallback_location, allow_other_host: allow_other_host, **options
```

(Note: the `**options` naming on passed keyword arguments.)

Reply:
I don't think Shopify has a strong preference, but I updated to the above 👍